### PR TITLE
Added env-vars key to example.site.yml

### PR DESCRIPTION
**Pull request by geek-merlin**
Followup for https://github.com/consolidation/site-process/pull/47

### CODE REVIEW

**Comment by greg-1-anderson** [comment]
I released consolidation/site-process version 2.1.0 with #47. We should probably update our composer.json to depend on ^2.1 before documenting here.

File: examples/example.site.yml [hunk 1]
```diff
@@ -182,6 +184,9 @@
 #     options: '-p 100'
 #   paths:
 #     drush-script: '/path/to/drush'
+#   env-vars:
+#     PATH: /bin:/usr/bin:${env.HOME}/.composer/vendor/bin
```
Comment (weitzman) [inline examples/example.site.yml]: Using `env.home` is ambiguous here. Some folks will think it refers to the remote system but it I think it refers to the local. We need to document or remove.

Comment (geek-merlin) [reply examples/example.site.yml]: (trying github suggestion feature...)
```suggestion
#     PATH: /bin:/usr/bin:/home/www-admin/.composer/vendor/bin
```

Comment (geek-merlin) [reply examples/example.site.yml]: Yes that part was plain wrong. Although i think that `$HOME` for the remote env var might work and make sense here, let's omit that until it's tested.

**Comment by geek-merlin** [comment]
Aah ok, we're committing composer.lock... OK rebased and pushed.
<details><summary>
$ composer require consolidation/site-process:^2.1
</summary><pre>
    Finished: success: 4, skipped: 0, failure: 0, total: 4
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 88 installs, 3 updates, 0 removals
  - Updating webflo/drupal-finder (1.1.0 => 1.2.0): Loading from cache
  - Updating nikic/php-parser (v4.2.1 => v4.2.2): Loading from cache
  - Installing consolidation/site-process (2.1.0): Loading from cache
  - Installing lox/xhprof (dev-master c64571f): Cloning c64571f892 from cache
  - Installing brumann/polyfill-unserialize (v1.0.3): Loading from cache
  - Installing typo3/phar-stream-wrapper (v2.1.2): Loading from cache
  - Installing paragonie/random_compat (v2.0.18): Loading from cache
  - Installing symfony/polyfill-php70 (v1.11.0): Loading from cache
  - Installing symfony/http-foundation (v3.4.27): Loading from cache
  - Installing symfony/translation (v3.4.26): Loading from cache
  - Installing symfony/validator (v3.4.26): Loading from cache
  - Installing symfony/serializer (v3.4.26): Loading from cache
  - Installing symfony/routing (v3.4.26): Loading from cache
  - Installing symfony/polyfill-php72 (v1.11.0): Loading from cache
  - Installing symfony/http-kernel (v3.4.26): Loading from cache
  - Installing symfony/class-loader (v3.4.26): Loading from cache
  - Installing psr/http-message (1.0.1): Loading from cache
  - Installing symfony/psr-http-message-bridge (v1.1.2): Loading from cache
  - Installing symfony/polyfill-iconv (v1.11.0): Loading from cache
  - Installing pear/pear_exception (v1.0.0): Loading from cache
  - Installing pear/console_getopt (v1.4.1): Loading from cache
  - Installing pear/pear-core-minimal (v1.10.7): Loading from cache
  - Installing pear/archive_tar (1.4.6): Loading from cache
  - Installing doctrine/lexer (v1.0.1): Loading from cache
  - Installing egulias/email-validator (2.1.7): Loading from cache
  - Installing guzzlehttp/psr7 (1.4.2): Loading from cache
  - Installing guzzlehttp/promises (v1.3.1): Loading from cache
  - Installing guzzlehttp/guzzle (6.3.3): Loading from cache
  - Installing doctrine/cache (v1.6.2): Loading from cache
  - Installing zendframework/zend-stdlib (3.0.1): Loading from cache
  - Installing zendframework/zend-escaper (2.5.2): Loading from cache
  - Installing zendframework/zend-feed (2.7.0): Loading from cache
  - Installing zendframework/zend-diactoros (1.4.1): Loading from cache
  - Installing symfony-cmf/routing (1.4.1): Loading from cache
  - Installing stack/builder (v1.0.5): Loading from cache
  - Installing masterminds/html5 (2.3.0): Loading from cache
  - Installing easyrdf/easyrdf (0.9.1): Loading from cache
  - Installing doctrine/inflector (v1.1.0): Loading from cache
  - Installing doctrine/collections (v1.3.0): Loading from cache
  - Installing doctrine/annotations (v1.2.7): Loading from cache
  - Installing doctrine/common (v2.6.2): Loading from cache
  - Installing asm89/stack-cors (1.2.0): Loading from cache
  - Installing squizlabs/php_codesniffer (3.4.2): Loading from cache
  - Installing drupal/coder (8.3.5): Cloning 35277fc867 from cache
  - Installing justinrainbow/json-schema (5.2.8): Loading from cache
  - Installing sebastian/recursion-context (3.0.0): Loading from cache
  - Installing sebastian/exporter (3.1.0): Loading from cache
  - Installing sebastian/diff (2.0.1): Loading from cache
  - Installing sebastian/comparator (2.1.3): Loading from cache
  - Installing doctrine/instantiator (1.2.0): Loading from cache
  - Installing phpspec/prophecy (1.8.1): Loading from cache
  - Installing symfony/phpunit-bridge (v3.4.30): Loading from cache
  - Installing symfony/css-selector (v3.4.30): Loading from cache
  - Installing sebastian/version (2.0.1): Loading from cache
  - Installing sebastian/resource-operations (1.0.0): Loading from cache
  - Installing sebastian/object-reflector (1.1.1): Loading from cache
  - Installing sebastian/object-enumerator (3.0.3): Loading from cache
  - Installing sebastian/global-state (2.0.0): Loading from cache
  - Installing sebastian/environment (3.1.0): Loading from cache
  - Installing phpunit/php-text-template (1.2.1): Loading from cache
  - Installing phpunit/phpunit-mock-objects (5.0.10): Loading from cache
  - Installing phpunit/php-timer (1.0.9): Loading from cache
  - Installing phpunit/php-file-iterator (1.4.5): Loading from cache
  - Installing theseer/tokenizer (1.1.3): Loading from cache
  - Installing sebastian/code-unit-reverse-lookup (1.0.1): Loading from cache
  - Installing phpunit/php-token-stream (2.0.2): Loading from cache
  - Installing phpunit/php-code-coverage (5.3.2): Loading from cache
  - Installing phar-io/version (1.0.1): Loading from cache
  - Installing phar-io/manifest (1.0.1): Loading from cache
  - Installing myclabs/deep-copy (1.9.1): Loading from cache
  - Installing phpunit/phpunit (6.5.14): Loading from cache
  - Installing mikey179/vfsstream (v1.6.7): Loading from cache
  - Installing jcalderonzumba/gastonjs (v1.2.0): Loading from cache
  - Installing behat/mink (dev-master a534fe7): Cloning a534fe7dac from cache
  - Installing jcalderonzumba/mink-phantomjs-driver (v0.3.3): Loading from cache
  - Installing instaclick/php-webdriver (1.4.5): Loading from cache
  - Installing behat/mink-selenium2-driver (1.3.x-dev 0a09c43): Cloning 0a09c43416 from cache
  - Installing symfony/dom-crawler (v4.2.11): Loading from cache
  - Installing symfony/browser-kit (v4.3.3): Loading from cache
  - Installing fabpot/goutte (v3.2.3): Loading from cache
  - Installing behat/mink-browserkit-driver (1.3.3): Loading from cache
  - Installing behat/mink-goutte-driver (v1.2.1): Loading from cache
  - Installing drupal/core (8.7.x-dev 5289b1a): Cloning 5289b1a751 from cache
  - Installing drupal/empty_theme (1.0.0): Loading from cache
  - Updating symfony/var-dumper (v3.4.26 => v4.3.3): Loading from cache
  - Installing drupal/devel (2.1.0): Loading from cache
  - Installing wamania/php-stemmer (1.2): Loading from cache
  - Installing drupal/alinks (1.0.0): Loading from cache
  - Installing vlucas/phpdotenv (v2.6.1): Loading from cache
</pre></details>

**Comment by weitzman** [comment]
Just occurred to me that setting /path/to/php would be genuinely useful for many. Could that go into the example?

**Comment by geek-merlin** [comment]
> Just occurred to me that setting /path/to/php in PATH would be genuinely useful for many. Could that go into the example?

Usually this is /usr/bin so it's in already.
(If that answer does not make sense to you then i don't get the question ;-) In that case you may want to use the gitbub suggest-change button.

**Comment by geek-merlin** [comment]
🎉!

**Comment by weitzman** [comment]
I was suggesting a path to a custom php that's not already in PATH.  it's a minor point. Merged.

**Pull request merged**